Earthfile

```diff
@@ -104,3 +102,4 @@ test-setup:
    RUN mix local.rebar --force
    RUN mix local.hex --force
    RUN mix deps.get
+   RUN MIX_ENV=test mix compile
```

Comment:
Would it work to move this to the `:test` target and change it to `MIX_ENV=test mix deps.compile`? Since this is currently in the shared setup, it will also be run before the NPM tests which I don't believe need it.

Reply:
Yep, seems like a good idea.

Reply:
@aaronrenner would you mind taking another look?

Reply:
The update looks great. I just want to run this locally first. I'll try to get you feedback in the next evening or two.

Reply:
@aaronrenner OK, thanks!